sdk/servicebus/service-bus/src/models.ts

```diff
@@ -83,7 +83,7 @@ export type SubQueue = "deadLetter" | "transferDeadLetter";
  * @interface CreateReceiverOptions
  * @template ReceiveModeT
  */
-export interface CreateReceiverOptions<ReceiveModeT extends ReceiveMode> {
+export interface ServiceBusReceiverOptions<ReceiveModeT extends ReceiveMode> {
```

Comment:
The jsdocs still refers to the old name. Instead of renaming there, please remove the `@interface CreateReceiverOptions` line altogether. In TS files, the `@interface` tag is not needed as that information can be inferred

Reply:
Same for ServiceBusSessionReceiverOptions